src/main/kotlin/com/epam/carapi/exception/NotFoundException.kt

```diff
@@ -0,0 +1,5 @@
+package com.epam.carapi.exception
+
+import org.springframework.http.HttpStatus
+
+open class NotFoundException(message: String) :ServiceException(HttpStatus.NOT_FOUND, message)
```

Comment:
I noticed that in some cases a colon have or doesn't have a space around when you extend/implement a class/interface. The common convention according to [the Kotlin documentation](https://kotlinlang.org/docs/inheritance.html) is to surround the colon by spaces. So it should look like the following ('_' means space):
`class A()_:_B()`
Even if there're no code style rules or conventions it's good to be consistent.